kPairs.py

```diff
@@ -0,0 +1,23 @@
+# Siimple 2 pointers approach to solve the problem, if current sum is greater than k then move j else move i and st j to last element of array else move i.
+# Time Complexity: O(n)
+# Space complexity: O(n)
+def func(arr,k):
```

Comment:
Please check your reasoning/logic here. Doesn't work fine.
https://leetcode.com/problems/k-diff-pairs-in-an-array/